string_case.go

```diff
@@ -0,0 +1,20 @@
+package microerror
+
+import (
+	"strings"
+	"unicode"
+)
+
+func toStringCase(input string) string {
```

Comment:
Would be nice to provide a comment for this function so we do not have to look at test to understand what is the purpose here.

Reply:
I am not sure what I should write here. The comment is either too simple to reflect what it really does to its fullest extend or to complicated to understand and then you look at the tests. Maybe suggest something when you really want to see a comment here.